e2e/cypress/support/api/preference.d.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/// <reference types="cypress" />
+
+// ***************************************************************
+// Each command should be properly documented using JSDoc.
+// See https://jsdoc.app/index.html for reference.
+// Basic requirements for documentation are the following:
+// - Meaningful description
+// - Specific link to https://api.mattermost.com
+// - Each parameter with `@params`
+// - Return value with `@returns`
+// - Example usage with `@example`
+// Custom command should follow naming convention of having `api` prefix, e.g. `apiLogin`.
+// ***************************************************************
+
+declare namespace Cypress {
+    interface Chainable<Subject = any> {
+
+        // *******************************************************************************
+        // Preferences
+        // https://api.mattermost.com/#tag/preferences
+        // *******************************************************************************
+
+        /**
+         * Save a list of the user's preferences.
+         * See https://api.mattermost.com/#tag/preferences/paths/~1users~1{user_id}~1preferences/put
+         * @param {object[]} preferences - List of preference objects
```

Comment:
nit: please update too.

```suggestion
         * @param {PreferenceType[]} preferences - List of preference objects
```

Reply:
Updated.